charts/producer-app-cleanup-job/templates/job.yaml

```diff
@@ -35,6 +35,9 @@ spec:
       {{- if .Values.tolerations }}
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.priorityClassName }}
```

Comment:
Not needed for clean up jobs

Reply:
I think you forgot to refresh

Reply:
I did and the comment disappeared because the line disappeared, but magically it still got submitted^^